services/wallet/router/pathprocessor/constants.go

```diff
@@ -13,10 +13,13 @@ var (
 
 const (
 	IncreaseEstimatedGasFactor = 1.1
+	SevenDaysInSeconds         = 604800
```

Comment:
I know that this number is accurate, would you consider showing the calculation instead of the full number. It makes accidentally changing this const much more difficult to do.

I don't have a strong opinion on this though, so if you prefer the full number that is completely fine with me.

```suggestion
	SevenDaysInSeconds         = 60 * 60 * 24 * 7
```